src/Notify/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+-   Added options to Notification
+-   Rename content from \_notify to title
```

Comment:
I suggest using code spans (i.e. wrapping in backticks) there for the actual keys